addons/project-notes-tabs/style.css

```diff
@@ -29,3 +29,7 @@
 .sa-selected {
   background-color: rgba(77, 151, 255, 0.1);
 }
+
+.project-description {
+  height: -webkit-fill-available;
```

Comment:
We support Firefox as well

Reply:
What does ff use?

Reply:
I'd bet `-moz`, but you'd need to do your research (and test it!)

Reply:
I don't have firefox sadly but I'll fix this.